src/workloads/execution/TimeSeriesGroupDateTrunc.yml

```diff
@@ -0,0 +1,113 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-execution"
+Description: |
+  Timeseries performance test
```

Comment:
If you are going to merge, can we get a more complete description, including what the workload is for, what it does, what the phases are doing.

Also some keyword search terms  under 'Keywords' attribute.